tests/libs/netcdf/tests-parallel/test_pnetcdf

```diff
@@ -14,6 +14,9 @@ rm=$RESOURCE_MANAGER
 testname="libs/NetCDF"
 delimiter="$RESOURCE_MANAGER/$LMOD_FAMILY_COMPILER/$LMOD_FAMILY_MPI"
 
+if [ ! -z "$SIMPLE_CI" ]; then
+    export OMPI_MCA_btl="self,tcp"
+fi
```

Comment:
Sorry, I have just seen this now. Is this still necessary? Even with the latest version of packages.

Reply:
Let's try without it!

Reply:
Yes, it was still failed before, let's see results.

Reply:
It is still needed - https://github.com/openhpc/ohpc/actions/runs/4445282348/jobs/7804302549?pr=1681

Reply:
Let me also test it locally. I think I saw it working without it.

Reply:
not sure what I did to get it work. doesn't work any more locally without that change